src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/HdcService.kt

```diff
@@ -61,5 +66,7 @@ class HdcService(
       val approvedForHdc = approvedIds.contains(bookingId)
       return (kind == HDC && approvedForHdc) || (kind != HDC && !approvedForHdc)
     }
+
+    fun isApprovedForHdc(bookingId: Long) = approvedIds.contains(bookingId)
```

Comment:
Slight threw me that this function and the new addition are both called `isApprovedForHdc` but can see that one is a function for the service and one for the HdcStatuses